actionmailer/test/base_test.rb

```diff
@@ -93,15 +93,15 @@ class BaseTest < ActiveSupport::TestCase
             "In-Reply-To" => "1234@mikel.me.com" }
     mail = BaseMailer.welcome(hash)
     assert_equal("SecretValue", mail["X-Special-Domain-Specific-Header"].decoded)
-    assert_equal("1234@mikel.me.com", mail["In-Reply-To"].decoded)
+    assert_match(/<?1234@mikel.me.com>?/, mail["In-Reply-To"].decoded)
   end
 
   test "can pass random headers in as a hash to headers" do
     hash = { "X-Special-Domain-Specific-Header" => "SecretValue",
             "In-Reply-To" => "1234@mikel.me.com" }
     mail = BaseMailer.welcome_with_headers(hash)
     assert_equal("SecretValue", mail["X-Special-Domain-Specific-Header"].decoded)
-    assert_equal("1234@mikel.me.com", mail["In-Reply-To"].decoded)
+    assert_match(/<?1234@mikel.me.com>?/, mail["In-Reply-To"].decoded)
```

Comment:
Can we change to `assert_equal "<1234@mikel.me.com>"` rather than making the brackets optional?

Reply:
it would fail on mail 2.7.1